insights/combiners/package_provides_java.py

```diff
@@ -40,6 +41,10 @@ class PackageProvidesJavaAll(LegacyItemAccess):
     """
 
     def __init__(self, package_provides_java):
+        deprecated(
```

Comment:
Please include information about the deprecation in the docstring so that it will be included in the documentation.

Reply:
Done